### PR TITLE
Add thumbnail support

### LINKED ISSUE #1116: Support thumbnail property, like web.dev
On web.dev you can set a `thumbnail` property if you don't want to repurpose your hero image for your blog card. The thumbnail will also be used as the social media image if no hero is provided. But d.c.c. doesn't support thumbnail and we probably should (since the request will surely come up, just as it did on web.dev). See also #1115

### PR DESCRIPTION
**Pull request by robdodson**
Fixes #1116 

Adds support for `thumbnail` front matter property.

### CODE REVIEW

File: site/en/blog/_example/index.md [hunk 1]
```diff
@@ -41,9 +41,18 @@ tags:
 # https://developer.chrome.com/docs/handbook/how-to/add-media/
 hero: 'image/BrQidfK9jaQyIHwdw91aVpkPiib2/EnMzOm0mBytBA3AzlCG6.png'
 
-# Required if there is a hero image
+# Optional
+# By default the hero image will be used for twitter and facebook cards, so if
+# you have a hero image then you don't need to provide a socialImage.
+# This should really only be used in the case where you don't want a hero image
+# but you DO want a twitter/facebook card image.
+# Social images should be at least 1920 x 960
```
Comment (samthor) [inline site/en/blog/_example/index.md:49]: ... something about "but we render them at 1521".

**Comment by samthor** [comment]
I'm fine with this but I don't really understand the distinction between `socialImage` and `thumbnail`.

**Comment by robdodson** [comment]
yeah on second thought, I'm going to change this to just be `thumbnail`. I'll make the necessary updates to cards as well.

File: site/_includes/partials/meta.njk [hunk 2]
```diff
@@ -28,13 +24,15 @@
 <meta property="og:type" content="website" />
 <meta property="og:url" content="{{ currentUrl }}" />
 
-{% if hero %}
+{% set socialImage = thumbnail or hero or site.defaultSocialImage %}
```
Comment (samthor) [inline site/_includes/partials/meta.njk:27]: I think I get it now: thumbnail "wins" for meta tags, and hero "wins" for actual content pages.

I think that's not incredibly obvious but I'm happy to LGTM anyway.

**Comment by robdodson** [comment]
@samthor did my PR introduce a LHCI breakage or was it already broken?

**Comment by samthor** [comment]
Apparently yes, a clean PR has no problems. I'm still concerned it's noise, though.

**Comment by samthor** [comment]
Is noise. Will submit this.

**Pull request merged**